aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -656,6 +718,20 @@ module aptos_framework::fungible_asset {
         move_to(&metadata_object_signer, supply);
     }
 
+    /// Ensure a known `FungibleStore` has `ConcurrentFungibleBalance`.
+    inline fun ensure_store_upgraded_to_concurrent_internal(
+        fungible_store_address: address,
+    ) {
+        if (!exists<ConcurrentFungibleBalance>(fungible_store_address)) {
+            let store = borrow_global_mut<FungibleStore>(fungible_store_address);
+            let balance = aggregator_v2::create_unbounded_aggregator();
```

Comment:
maybe we should support initialize with a value.

Reply:
yeah, if we started with that, we could've had one native and one utility functions. Seems like we will have to have two native functions now :/